### DISCUSSION TITLE
Multiple-level structure list: Drag and drop sortable tree

### DISCUSSION THREAD

**Discussion by lovishhamal** [shadcn-ui/ui#3281] [Ideas]
I would like to propose the implementation of a multiple-level sortable list feature in Shadcn-ui. This feature would allow users to organize items within a list by dragging and dropping them into different positions, similar to how single sortable lists currently function. However, this feature would extend the functionality to support multiple lists that can be sorted independently.

With this feature, users could seamlessly prioritize and manage items across different categories or projects within the shadcn-ui interface. Whether organizing components, tasks, or other elements, this functionality would enhance usability and productivity for users navigating complex workflows.

**Comment by boudewijnbout** [comment]
I would love to see this too!
